readme.md

```diff
@@ -8,20 +9,20 @@ Hieronder globale informatie over het opzetten van een lokale ontwikkelomgeving.
 ## Draai de code lokaal
 
 - Installeer WAMP
-- Installeer Composer
-- `$ composer create-project --prefer-dist laravel/laravel=5.1.* aas2`
+- Optioneel: installeer Composer globaal (maar je kunt ook `$ php composer.phar <command>` gebruiken binnen deze repo)
 - Clone dit project
-- Kopieer de bestanden van dit project over de gegenereerde bestanden in de map `aas2` (die in de map van je WAMP-installatie staat)
-- `$ composer update` haalt de dependencies binnen van de source code
-
+- `$ composer install` haalt de dependencies binnen van de source code en maakt een lokale `.env` file aan
+- Maak in WAMP een Virtual Host aan die verwijst naar de `public` map van dit project
+- Open het aangemaakte adres in je favoriete browser
 
 _Protip: [Laravel IDE helper generator](https://github.com/barryvdh/laravel-ide-helper) zou vermoedelijk je IDE significant kunnen verbeteren, o.a. code completion, linking van variablen in verschillende soorten bestanden._
 
 ## Database verbinden
+
 - Maak een nieuwe gebruiker met eigen database aan in PhpMyAdmin
 - Schrijf de inloggegevens in `.env` in de projectfolder
-- Vul de tabel met: `$ php artisan migrate --seed`  
+- Maak en vul de tabellen met: `$ php artisan migrate --seed`  
 
 ## Testdata updaten
 
-`$ php artisan migrate:refresh --seed`
+`$ php artisan migrate:fresh --seed` verwijdert alle tabellen, maakt ze opnieuw aan en vult ze met dummy data
```

Comment:
Ik kende fresh nog niet, dat is echt zwaar handig.

Reply:
Ja superhandig. Heb ik gezien bij een webcast van de maker van Laravel, Taylor Otwell. Die heeft er zelfs een bash alias voor gemaakt: `mfs`.

Ik zie nog geen expliciete approval van de PR, komt dat omdat je de inhoudelijke review nog moet doen of heb je die knop gewoon niet gevonden 😛

Reply:
Ik was nog bezig ;)